hpx/runtime/get_ptr.hpp

```diff
@@ -173,21 +220,69 @@ namespace hpx
     ///            requesting locality. Otherwise the function will raise and
     ///            error.
     ///
+    /// \note      The component instance the returned pointer refers to can
+    ///            not be migrated as long as there is at least one copy of the
+    ///            returned shared_ptr alive.
+    ///
     /// \note      As long as \a ec is not pre-initialized to \a hpx::throws this
     ///            function doesn't throw but returns the result code using the
     ///            parameter \a ec. Otherwise it throws an instance of
     ///            hpx::exception.
     ///
     template <typename Component>
     std::shared_ptr<Component>
-    get_ptr(launch::sync_policy, naming::id_type const& id,
+    get_ptr(launch::sync_policy p, naming::id_type const& id,
```

Comment:
Is the naming of the `launch::sync_policy` parameter on purpose? It might trigger a compiler warning.